packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -275,7 +276,7 @@ export async function editSDKConnection(
   if (needsProxyUpdate) {
     // Purge CDN if used
     await purgeCDNCache(connection.organization, [connection.key]);
-
+    await purgeEdgeRemoteEvalSDKKeys(connection.organization, [connection.key]);
```

Comment:
We have a ton of different triggers that all fire when an SDK Payload changes:
- CDN Purge (Fastly)
- Legacy SDK Webhooks
- Cloud Proxy
- Per-connection Proxy
- Edge Remote Eval
- Global SDK Webhook (coming soon)
- Per-connection Webhook (coming soon)

I think we should take a step back and try to abstract out common code between all of these